st2auth/st2auth/controllers/v1/auth.py

```diff
@@ -47,6 +49,25 @@ def post(self, request, **kwargs):
         elif cfg.CONF.auth.mode == 'standalone':
             return self._handle_standalone_auth(request=request, **kwargs)
 
+    @jsexpose(arg_types=[str])
+    def get_one(self, token):
```

Comment:
Since this method validates the token, I would prefer to call it and such and if possible (if that doesn't make integration too harder) also utilize POST method.

So something like: `POST /v1/tokens/validate`.